projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-manager.service.spec.ts

```diff
@@ -100,10 +104,10 @@ describe('DataLayerManagerService', () => {
       );
     });
 
-    it('should emit $timelineRange when timeline scale limits change', waitForAsync(() => {
```

Comment:
I cleaned up unnecessary waitForAsync, which is not needed with jasmine-marbles test scheduler